### DISCUSSION TITLE
Version 2.5.3

### DISCUSSION THREAD

**Discussion by JonShibata** [pythonnet/pythonnet#2301] [Q&A]
I updated to the latest package which broke some scripts written by a coworker who has left. Before updating I pulled the version information and it says 2.5.3. I would like to revert to that version since I no longer have support for updating the scripts, but I don't see it here? Has that version been removed? Is there a way I can get it?

**Comment by filmor** [comment]
2.5.3 doesn't exist. If your codebase refers to that, it must be a custom build.
